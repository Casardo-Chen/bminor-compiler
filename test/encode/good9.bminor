"hello \0x01 world"