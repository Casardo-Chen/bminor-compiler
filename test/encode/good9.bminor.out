Input String    : "hello \0x01 world"
Decoded String    : hello  world
Output String    : "hello \0x01 world"
